### ISSUE TITLE
Support v-on like eval syntax

### ISSUE THREAD

**Issue by cco3** [cadgerfeast/vue-konami#5]
It would be cool if v-konami functioned more like the v-on syntax, where you can include snippets of javascript rather than just a method name.  For example, instead of doing this:

```
<div v-konami="myMethod"/>
<div v-if="konami"/>
...
methods: {
  myFunc() {
    konami = true
  }
}
```

You could do this:
```
<div v-konami="konami = true"/>
<div v-if="konami"/>
```

To that end, it seems like it might be simplest if you could just create a "dynamic event".  Then you could do `@[konami]="..."`

https://vuejs.org/v2/api/#v-on

**Comment by cadgerfeast** [comment]
Hi @cco3,

Unfortunately, VueJS@2 does not support literal expressions.
That said, I published a new version: **vue-konami@1.0.9** that supports it via a workaround, It dispatches a **konami** event, that you can listen on the element, you can use it this way:

``` html
<!-- Method -->
<div v-konami @konami="showEasterEgg = !showEasterEgg"></div>
<div v-if="showEasterEgg">My super EasterEgg</div>
<script>
export default {
  name: 'Konami',
  data () {
    return {
      showEasterEgg: false
    }
  }
}
</script>
```

**Comment by cco3** [comment]
Works, thanks!
